fili-core/src/test/groovy/com/yahoo/bard/webservice/metadata/BaseDataSourceMetadataSpec.groovy

```diff
@@ -105,63 +151,51 @@ class BaseDataSourceMetadataSpec extends Specification {
             getPartitionNum() >> 1
         }
 
-        Integer binversion1 = 9
-        long size1 = 1024
-        long size2 = 512
-
-        segment1 = new DataSegment(
-                tableName,
-                interval1,
-                version1,
-                null,
-                dimensions123,
-                metrics123,
-                partition1,
-                binversion1,
-                size1
-        )
-
-        segment2 = new DataSegment(
-                tableName,
-                interval1,
-                version2,
-                null,
-                dimensions123,
-                metrics123,
-                partition2,
-                binversion1,
-                size2
-        )
-
-        segment3 = new DataSegment(
-                tableName,
-                interval2,
-                version1,
-                null,
-                dimensions123,
-                metrics123,
-                partition1,
-                binversion1,
-                size1
-        )
-
-        segment4 = new DataSegment(
-                tableName,
-                interval2,
-                version2,
-                null,
-                dimensions123,
-                metrics123,
-                partition2,
-                binversion1,
-                size2
-        )
-
-        segments = [segment1, segment2, segment3, segment4]
-
-    }
-
-    def shutdownSpec() {
-        DateTimeZone.setDefault(currentTZ)
+        [
```

Comment:
Personally, I prefer to use an explicit `return` statement in functions/methods that are longer than one or two lines. Makes it clearer what you're doing with that last line.